src/mxcp/runtime/__init__.py

```diff
@@ -192,11 +202,14 @@ def _run_init_hooks():
             logger.error(f"Error in init hook {hook.__name__}: {e}")
 
 
-def _run_shutdown_hooks():
-    """Run shutdown hooks"""
+def run_shutdown_hooks():
+    """Run all registered shutdown hooks."""
     for hook in _shutdown_hooks:
         try:
             logger.info(f"Running shutdown hook: {hook.__name__}")
             hook()
         except Exception as e:
-            logger.error(f"Error in shutdown hook {hook.__name__}: {e}") 
\ No newline at end of file
+            logger.error(f"Error in shutdown hook {hook.__name__}: {e}")
```

Comment:
It was already the case that we'd only log an error in case a hook would fail. I can imagine one would want to not miss that (e.g. if the connection to a backend fails within the init hook, just stop the MCP). Wondering why not failing.